src/index.js

```diff
@@ -23,29 +23,50 @@ function refreshTable(newData){
     const totalEm = document.querySelector('.total')
     totalEm.innerText = 'Total: ' + totalNumbers
 
-    contributors = contributors.sort( (a, b) => {
-        if ( a.mergedPRsNumber < b.mergedPRsNumber ){
+    contributors = contributors.sort((a, b) => {
+        var first, second, third // criteria is specified here
```

Comment:
@im-adithya what you think of naming these variables as preference1, preference2, and preference3.? Or something which may be easier for someone new to understand the logic behind it.

Reply:
Sure! Will do it.

Reply:
@djcruz93 I've uploaded the video now. Please review it :)